packages/playwright-ct-vue/index.d.ts

```diff
@@ -59,27 +63,17 @@ interface MountResultJsx extends Locator {
 
 export interface ComponentFixtures {
   mount(component: JSX.Element): Promise<MountResultJsx>;
-  mount<HooksConfig extends JsonObject>(
-    component: any,
-    options?: MountOptions<HooksConfig, Record<string, unknown>>
-  ): Promise<MountResult<Record<string, unknown>>>;
-  mount<
-    HooksConfig extends JsonObject,
-    Props extends Record<string, unknown> = Record<string, unknown>
-  >(
-    component: any,
-    options: MountOptions<HooksConfig, never> & { props: Props }
-  ): Promise<MountResult<Props>>;
+  mount<HooksConfig extends JsonObject, Component = unknown>(
```

Comment:
do you think this will break existing clients? or do we want to break them with these stricter types?